admin/services/prepare-check.service.js

```diff
@@ -87,8 +87,9 @@ function constructPreparedCheck (check) {
     checkCode: check.pupil.checkCode,
     config: check.config,
     createdAt: moment(),
-    pinExpiresAt: moment(check.pupil.pinExpiresAt),
-    pupil: R.omit(['id', 'checkFormAllocationId', 'pinExpiresAt'], check.pupil),
+    pinExpiresAtUtc: moment(check.pupil.pinExpiresAtUtc),
+    pinValidFromUtc: moment(check.pupil.pinValidFromUtc),
+    pupil: R.omit(['id', 'checkFormAllocationId', 'pinExpiresAtUtc', 'pinValidFromUtc'], check.pupil),
```

Comment:
@jon-shipley I believe we need to omit the new column `pinValidFromUtc` as we don't want this to exist in the pupil object of the prepared check.